### DISCUSSION TITLE
Small code change to allow fallback/default policies

### DISCUSSION THREAD

**Discussion by shawnheide** [laravel/nova-issues#6321] [Ideas & Feature Requests]
I've been trying to implement default policies and noticed that Nova handles policies slightly different to the Laravel framework. In the policy checks in Laravel, they use `is_callable` to check if a [policy class has a method](https://github.com/laravel/framework/blob/954c226343f438c453019019c00eb3ae54923b2c/src/Illuminate/Auth/Access/Gate.php#L806C44-L806C45). Nova uses `method_exists` which doesn't allow the `__call` method to be used in a policy class.

In order to allow for things like fallback or default policies, Nova would need to replace the `method_exists` calls in Authorizable.php with `is_callable`. I tested this out locally and it seems to work without any issues (so far). Here is the regex for replacing if you are curious: replace all `method_exists\(\$gate, (.+?)\)` to `is_callable([$gate, $1])` in the Authorizable trait in Nova. 

Let me know if you have any questions. Thanks for your time in considering this change.

Related issue: #1147

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
